app/build.gradle

```
@@ -97,4 +108,11 @@ dependencies {
     testImplementation 'org.threeten:threetenbp:1.4.0'
     testImplementation 'com.squareup.retrofit2:retrofit-mock:2.6.0'
     testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4"
+
+    //Hilt Testing
+    androidTestImplementation "com.google.dagger:hilt-android-testing:2.44"
+    kaptAndroidTest "com.google.dagger:hilt-android-compiler:2.44."
```

looks like there's an extra dot (.) at the end here:

```suggestion
    kaptAndroidTest "com.google.dagger:hilt-android-compiler:2.44"
```